README.md

```diff
@@ -1,44 +1,76 @@
 # ioFog Platform
 
-ioFog Platform provides a means by which to orchestrate and test ioFog deployments on various infrastructure.
+ioFog Platform provides a means by which to spin up and deploy an Eclipse ioFog stack on GKE and Packet.
 
-## iofog CLI
+# Requirements
 
-Automation of ioFog Platform is handled through the ioFog CLI. The CLI consumes a number of plugins for the deployment infrastructure and services.
+* Gcloud SDK
+* Terraform (v0.11.x)
+* Ansible
+* gcloud
+* Kubectl
+* Iofogctl
 
-![img not found](https://raw.githubusercontent.com/eclipse-iofog/platform/develop/cli.png)
+You can run `./bootstrap.sh` in order to download thoses dependencies.
 
-To create a plugin, you must implement a Python module in the plugins folder.
+You will also require the following environment variables
+```sh
+export PACKAGE_CLOUD_TOKEN=<package_cloud_token> # If you need to download private packages from packagecloud
+export PACKET_AUTH_TOKEN=<packet_auth_token> # If you want to deploy agents on Packet
+export GOOGLE_APPLICATION_CREDENTIALS=<path-to-json>
+```
+You can edit the file `./my_credentials.sh` to provide your keys.
```

Comment:
Do we need a separate file? the bash script can check if these values exists and prompt if not, since Packet is optional - can be a user input yes or no to proceed without that env var set.

if you think we the separate file is needed, add it to gitignore now, so it dont get accidentally pushed with secrets.

Reply:
The file `my_credentials.sh` gets generated by `./bootstrap.sh` based on `./scripts/credentials.template.sh` so it is not part of the version controlled files. But I will add it to .gitignore to avoid any mistake.